header/log_includes/compress.h

```diff
@@ -1,6 +1,4 @@
-
```

Comment:
File name is too generic, rename to something specific to functionality, similar to cpp file.
Like if  ABC.cpp is source file, then its header file should be ABC.h
FYI: https://google.github.io/styleguide/cppguide.html